tests/jsx_escape/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -11,3 +11,38 @@ exports[`test escape.js 2`] = `
 <div>&lt;</div>;
 "
 `;
+
+exports[`test nbsp.js 1`] = `
+"many_nbsp = <div>&nbsp; &nbsp; </div>
+single_nbsp = <div>&nbsp;</div>
+many_raw_nbsp = <div>   </div>
+
+
+amp = <span>foo &amp; bar</span>
+raw_amp = <span>foo & bar</span>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+many_nbsp = <div>&nbsp; &nbsp; </div>;
+single_nbsp = <div>&nbsp;</div>;
+many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
+amp = <span>foo &amp; bar</span>;
+raw_amp = <span>foo & bar</span>;
+"
+`;
+
+exports[`test nbsp.js 2`] = `
+"many_nbsp = <div>&nbsp; &nbsp; </div>
+single_nbsp = <div>&nbsp;</div>
+many_raw_nbsp = <div>   </div>
+
+
+amp = <span>foo &amp; bar</span>
+raw_amp = <span>foo & bar</span>
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+many_nbsp = <div>&nbsp; &nbsp; </div>;
+single_nbsp = <div>&nbsp;</div>;
+many_raw_nbsp = <div>&nbsp;&nbsp;&nbsp;</div>;
```

Comment:
note this makes it impossible to write a raw nbsp within JSX, which I personally view as a good thing.